Source/ASButtonNode.mm

```diff
@@ -18,6 +21,24 @@
 #import <AsyncDisplayKit/ASImageNode.h>
 #import <AsyncDisplayKit/ASInternalHelpers.h>
 
+#if YOGA
+static void ASButtonNodeResolveHorizontalAlignmentForStyle(ASLayoutElementStyle *style, ASStackLayoutDirection _direction, ASHorizontalAlignment _horizontalAlignment, ASStackLayoutJustifyContent _justifyContent, ASStackLayoutAlignItems _alignItems) {
```

Comment:
Could this be pulled out into a category or subclass?

Reply:
Done